presentation/build.gradle

```diff
@@ -116,7 +117,7 @@ android {
         dev {
             dimension "version"
             versionNameSuffix "-dev"
-            buildConfigField "String", SERVER_URL, "\"https://dev.api.forus.io/\""
+            buildConfigField "String", SERVER_URL, "\"https://test.api.forus.rminds.dev/\"" //\"https://dev.api.forus.io/\""
```

Comment:
which will it use?

Reply:
For dev it will use rminds  temporary domain for dev:  https://test.api.forus.rminds.dev
If needed, i can remove it